app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -291,8 +291,13 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
             context.components.useCases.tabsUseCases,
             context.components.useCases.contextMenuUseCases,
             view,
-            FenixSnackbarDelegate(view)
+            FenixSnackbarDelegate(view),
+            getDefaultDownloadPath()
         ) + ContextMenuCandidate.createOpenInExternalAppCandidate(requireContext(),
             contextMenuCandidateAppLinksUseCases)
     }
+
+    private fun getDefaultDownloadPath(): String {
+        return requireActivity().settings().downloadPath
```

Comment:
It this getting the latest updated value? I was experiencing issues using the `setting` class see https://github.com/mozilla-mobile/fenix/issues/13378. I was forced to use : `PreferenceManager.getDefaultSharedPreferences(context)` to get the latest value

Reply:
I was thinking that maybe `getDefaultDownloadDirectoy` must clear, I think we should do the same on ac part :)